modules/ROOT/pages/clauses/with.adoc

```diff
@@ -234,3 +234,56 @@ As the limit is larger than the total number of remaining rows, all rows are ret
 | 6
 |Rows: 4
 |===
+
+[[with-unique-results]]
+== Unique results
+
+`DISTINCT` retrieves only unique rows for the columns that have been selected for projection.
+
+.Query
+[source, cypher]
+----
+MATCH (n)-[:KNOWS]->(p)
+WITH DISTINCT p.name AS name
+RETURN name
+----
+
+`'George'` is returned by the query, but only once (without the `DISTINCT` operator it would have been returned twice because there are two `KNOWS` relationships going to it):
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| name
+| "Bossman"
+| "George"
+| "Anders"
+d|Rows: 3
+|===
+
+
+[[with-all-results]]
+== WITH ALL
+
+Projecting all results can also be accomplished by explicitly including `ALL` in the `WITH`.
+
```

Comment:
```suggestion
[role=label--new-2025.05]
[[with-all-results]]
== WITH ALL

Projecting all results can also be accomplished by explicitly including `ALL` in the `WITH`.
`WITH ALL` was introduced as part of Cypher's xref:appendix/gql-conformance/index.adoc[], and using it is functionally the same as using simple `WITH`.
```

Reply:
It isn't introduced for GQL compliance, WITH doesn't exist in GQL, it was introduced for consistency

Reply:
Ah yes, sorry!